mh/pat.py

```diff
@@ -1095,6 +1095,8 @@ def sendAnsBinaryHead(self, binary_type, seq):
          - Spanish: 0x41, 0x3d, 0x41, 0x3e, 0x3f, 0x40
         """
         binary = PAT_BINARIES[binary_type]
+        if callable(binary["content"]):
+            binary["content"] = binary["content"]()
```

Comment:
Since dictionaries are using references, I think you're overriding the function with its value so it won't be called ever again.
```python
content = binary["content"]
if callable(content):
    content = content()
```

Reply:
You're right, I've changed it to a way that does not override the value in the dictionary.

![image](https://user-images.githubusercontent.com/24629810/194725875-d24a97f5-7e19-4c5e-907b-cf4c8a82c834.png)